lesson-11/src/controllers/users.ts

```diff
@@ -1,8 +1,12 @@
 import { Request, Response } from "express";
 import * as userService from "../services/users";
 
-export const getAllUsers = (_req: Request, res: Response) => {
-  const users = userService.getAllUsers();
+export const getAllUsers = (req: Request, res: Response) => {
```

Comment:
query параметри теж можна типізувати
```suggestion
export const getAllUsers = (req: Request<{}, {}, {}, Record<string, string | undefined>>, res: Response) => {
```